shared_model/interfaces/permissions.hpp

```diff
@@ -96,9 +97,13 @@ namespace shared_model {
      public:
       PermissionSet();
       PermissionSet(std::initializer_list<Perm> list);
+      explicit PermissionSet(const std::string &bitstring);
 
-      size_t size() const;
+      std::string toBitstring() const;
+
+      static size_t size();
       PermissionSet &reset();
+      PermissionSet &set();
```

Comment:
Please add comments to methods

Reply:
I better do that in following pr, if you don't mind

Reply:
Sure